lib/dal/tests/integration_test/frame.rs

```diff
@@ -464,13 +477,21 @@ async fn simple_frames(ctx: &mut DalContext) {
 #[test]
 async fn output_sockets_can_have_both(ctx: &mut DalContext) {
     // create an even frame
-    let even_frame = create_component_for_schema_name(ctx, "large even lego", "even").await;
+    let even_frame = create_component_for_schema_name(ctx, "large even lego", "even")
+        .await
+        .expect("could not create component");
 
-    let _ = even_frame
+    even_frame
         .set_type(ctx, ComponentType::ConfigurationFrameDown)
-        .await;
-    let odd_component = create_component_for_schema_name(ctx, "large odd lego", "odd1").await;
-    let _ = odd_component.set_type(ctx, ComponentType::Component).await;
+        .await
+        .expect("could not set type");
```

Comment:
Now, we no longer swallow errors when setting type in frame tests. This is one example.